src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static ColorSupplier colorSupplier = new ColorSupplier();
+    private static Random random = new Random();
+    private static final int MAXIMUM_POSSIBLE_NUMBER = 100;
+
+    public static Ball getRandomBall() {
```

Comment:
Let it be non-static. See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-use-static-methods-in-your-solution).